app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -50,7 +50,7 @@
     private BluetoothAdapter bluetoothAdapter;
     private IBluetoothHandler bluetoothHandler;
     private Set<BluetoothDevice> pairedDevices;
-    private boolean isMaster = false;
+    private boolean isMaster;
```

Comment:
I think its is always better to explicitly set this value, even the most experienced programmer can make an error and interpret this wrongly...